Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 
+gem 'cocoapods'
```

Comment:
This shouldn't be needed. `cocoapods` is already a dependency inside the [test fixture project Gemfile](https://github.com/bugsnag/bugsnag-cocoa/blob/add-maze-harness/features/fixtures/ios-swift-cocoapods/Gemfile).